### ISSUE TITLE
Could  not create task

### ISSUE THREAD

**Issue by castn** [reimersoftware/tex-gradle-plugin#4]
Currently, I am trying to set up this plugin. So I created an empty Gradle project using Kotlin. I placed a test.tex file in the root directory next to `gradle.build.kts`

My build.gradle file looks like this:
```
import dev.reimer.tex.gradle.plugin.task.TexCompile

plugins {
    id("dev.reimer.tex") version "0.3.4"
}

tasks.register<TexCompile>("buildTexFile") {
    source("test.tex")
}
```

When trying to run this, I get the following error:
```
Could not create task ':buildTexFile'.
> Could not create task of type 'TexCompile'.
   > Could not create task ':generateResourcesForBuildTexFile'.
      > class dev.reimer.tex.gradle.plugin.internal.FilteredFileTree overrides final method org.gradle.api.internal.file.AbstractFileCollection.visitStructure(Lorg/gradle/api/internal/file/FileCollectionStructureVisitor;)V
```

What am I missing? I am using the latest version from the Gradle repo, but I have seen, that you updated it sometime ago.

**Comment by janheinrichmerker** [comment]
Is this issue fixed by #5?

**Comment by castn** [comment]
I think so, didn't have the issue again
